### ISSUE TITLE
Sharing information from map fragment

### ISSUE THREAD

**Issue by heysadboy** [fossasia/open-event-android#1659]
**Actual Behaviour**

Currently there is no functioning of sharing information from map fragment.

**Expected Behaviour**

That should be available from map.

**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

Milestone 5

**Comment by iamareebjamal** [comment]
@heysadboy What's the status?

**Comment by heysadboy** [comment]
@iamareebjamal I am on to other project, it would be great if you could assign this to other developers.

**Comment by iamareebjamal** [comment]
OK @shreyasnbhat @Shailesh351, any one of you want to take this up?

**Comment by Shailesh351** [comment]
I don't understand why we need sharing from map fragment ?

**Comment by shreyasnbhat** [comment]
@heysadboy What do you intend to share from the MapFragment? Could you be a bit more clear?

**Comment by heysadboy** [comment]
@shreyasnbhat @iamareebjamal @Shailesh351 Earlier there was a sharing option in the app from map to share the location but it was actually not that much of use in my view. After I added the searching facility, I removed that and this issue was to add it along with searching. 
I think the purpose of map is fulfilling. Should we close this? I think it was suggested during the merging of that feature to make this issue. @the-dagger What's your view?
